me.id.webverify/app/src/main/java/me/id/meidwebverify/MainActivity.java

```diff
@@ -1,88 +1,71 @@
 
 import me.id.webverifylib.IDmeWebVerify;
 
-public class MainActivity extends ActionBarActivity
-{
-    private IDmeWebVerify webVerify;
-    private String clientID = null;
-    private String redirectUri = null;
-    private boolean returnProperties = true;
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState)
-    {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+public class MainActivity extends ActionBarActivity {
+  private IDmeWebVerify webVerify;
+  private String clientID = null;
+  private String redirectUri = null;
+  private boolean returnProperties = true;
 
-        Button btnVerify = (Button) findViewById(R.id.btnVerify);
-
-        btnVerify.setOnClickListener(new View.OnClickListener()
-        {
-            @Override public void onClick(View v)
-            {
-                Verify();
-            }
-        });
-    }
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
 
-    /**
-     * Method that Starts the Verification Process.
-     */
-    public void Verify()
-    {
-        Spinner spnRoute = (Spinner) findViewById(R.id.spnRoute);
-        Spinner propRoute = (Spinner) findViewById(R.id.spnProperties);
-        String affiliationType = "";
+    Button btnVerify = (Button) findViewById(R.id.btnVerify);
+    btnVerify.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        Verify();
+      }
+    });
+  }
 
-        returnProperties = propRoute.getSelectedItem().toString().equals("Yes");
+  /**
+   * Method that Starts the Verification Process.
+   */
+  public void Verify() {
+    Spinner spnRoute = (Spinner) findViewById(R.id.spnRoute);
+    Spinner propRoute = (Spinner) findViewById(R.id.spnProperties);
+    String affiliationType = "";
 
-        if (spnRoute.getSelectedItem().toString().equals("Military"))
-        {
-            affiliationType = IDmeWebVerify.MILITARY;
-        }
-        else if (spnRoute.getSelectedItem().toString().equals("Student"))
-        {
-            affiliationType = IDmeWebVerify.STUDENT;
-        }
-        else if (spnRoute.getSelectedItem().toString().equals("Teacher"))
-        {
-            affiliationType = IDmeWebVerify.TEACHER;
-        }
-        else if (spnRoute.getSelectedItem().toString().equals("First Responder"))
-        {
-            affiliationType = IDmeWebVerify.FIRST_RESPONDER;
-        }
-        else if (spnRoute.getSelectedItem().toString().equals("Government")) {
-            affiliationType = IDmeWebVerify.GOVERNMENT;
-        }
+    returnProperties = propRoute.getSelectedItem().toString().equals("Yes");
 
-        webVerify = new IDmeWebVerify(clientID, redirectUri, affiliationType, this, returnProperties);
-        webVerify.StartWebView();
+    if (spnRoute.getSelectedItem().toString().equals("Military")) {
```

Comment:
yes, it'll return null if there is nothing selected, but the adapter always has elements, they are defined in the string resource file. It's the example application, how should be it's behavior if the variable is null?